### ISSUE TITLE
SSLOffloading options disappeared

### ISSUE THREAD

**Issue by smacz42** [PiBa-NL/pfsense-haproxy-package-doc#20]
There are no longer any ssloffloading options in frontends as of `0.61_10`.

When checking the SSLOffload checkbox on a frontend, it automatically gets assigned the first cert in pfsense's cert manager, regardless of what that cert is (Mine got asssigned a client VPN cert... a.k.a., NOT a "server" cert).

I tried uninstalling, rebooting, and reinstalling the package and the same behavior persists.

This was on pfsense 2.6.0 and 2.7.0.

![2023-07-07-01-15-16-Services_ HAProxy_ Frontend_ Edit - protense admin andrewcz com](https://github.com/PiBa-NL/pfsense-haproxy-package-doc/assets/10110994/4aa59c12-ccd2-40b9-b9d2-4cd6522d99bc)

**Comment by mipereboom** [comment]
Can confirm. 

Also unable to download HAProxy from the available packages without updating to 2.7.0, so I'm stuck until this gets fixed.

**Comment by gustovshikov** [comment]
I have the same issue on pfsense 2.7.0. Tried haproxy-devel package but still the same problem.

**Comment by smacz42** [comment]
FWIW... when I disabled the [Dark Reader](https://darkreader.org) browser extension for that site, the SSL offloading section came back.

The reason I tried that was that I was looking at the source and did in fact see that section, but it wasn't being shown for some reason. I don't know if there's some oddities in the html that makes it odd for browser plugins, or at least Dark Reader.

So, for me (at least) this is solved. However, I'm not sure about for the two reports above.

**Comment by gustovshikov** [comment]
I was using dark reader as well. Disabling it fixed the issue.

**Comment by andrew-flatters-bliss** [comment]
@smacz42 o wow. Didn't even think about DarkReader as a possibility. Cheers! That solved my issue too.

I've also sent the author of this extension a email with feedback with this GitHub issue so hopefully can solve their end too.

**Comment by merc4derp** [comment]
I'm using dark reader and the ssl offloading section appears just fine on both chrome and firefox. 
Obviously dark reader inactive for pfsense's web configurator since it has its own dark theme.

**Comment by andrew-flatters-bliss** [comment]
@merc4derp yup what I switched over to since disabling Dark reader - didn't even realise it was on to be fair.
